README.md

```diff
@@ -35,6 +35,12 @@ This mode is intended for developers who are willing to accept the risk of using
 
 > NOTE: Keep clearly in mind that the experimental mode fundamentally changes the Azle Wasm binary. It is not guaranteed to be secure or stable in API changes or runtime behavior. If you enable the experimental mode, even if you only use APIs from the stable mode, you are accepting a higher risk of bugs, errors, crashes, security exploits, breaking API changes, etc.
 
```

Comment:
Explore automatically doing this npm install, but with the correct hash pulled from azle's package.json, and install it into the developer's package.json

Reply:
Remove this from the README

Reply:
Done, but I'm not really happy with it, we should explore it a little more

Reply:
done